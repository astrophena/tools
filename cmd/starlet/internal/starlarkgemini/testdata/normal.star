result = gemini.generate_content(
    contents = ["Write a story about very kind heiress and her maid."],
)

print(result[0][0])
