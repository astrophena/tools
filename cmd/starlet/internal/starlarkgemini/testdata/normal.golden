The sun, a golden coin, peeked over the horizon, casting long shadows across the sprawling estate of Willow Creek. Inside the grand manor, Lady Amelia, heiress to the vast Willow Creek fortune, was already awake, her heart brimming with a gentle, unassuming kindness that belied her status.

Amelia wasn't like other heiresses. She didn't crave lavish balls or designer gowns. Her joy resided in simple things: the soft rustle of leaves in the wind, the scent of freshly baked bread, and the companionship of her maid, Eliza.

Eliza, with her warm brown eyes and a smile that could light up a room, was more than just a maid. She was Amelia's confidante, her closest friend. They shared laughter, secrets, and a deep bond forged in the crucible of mutual respect and affection.

One autumn afternoon, as the leaves danced a fiery ballet outside, Amelia found Eliza in the kitchen, her brow furrowed over a worn book. "What troubles you, Eliza?" Amelia asked, her voice as gentle as the breeze.

"It's my brother, Milly," Eliza sighed, her voice tinged with worry. "He's been ill for weeks, and the doctor says he needs a special tonic that we can't afford."

A shadow crossed Amelia's face, but it quickly faded. "We'll find a way, Eliza," she said, her voice firm with resolve. "He will be well again."

True to her word, Amelia discreetly contacted a renowned apothecary, sending him a generous sum and a detailed description of Eliza's brother's ailment. The tonic arrived within a week, wrapped in velvet and smelling of herbs and hope.

Eliza, overwhelmed with gratitude, wept tears of joy. "You shouldn't have done this, Milly," she said, her voice choked with emotion.

Amelia, her face radiating warmth, replied, "Eliza, you are my family. Your happiness is my happiness."

As Eliza tended to her ailing brother, Amelia, who was known for her philanthropy, anonymously set up a fund to help other impoverished families in need. Her kindness, like a silent river, flowed out, touching countless lives and leaving behind a legacy of compassion.

Years passed, and the bond between Amelia and Eliza grew even stronger. They navigated life's joys and sorrows together, their friendship a beacon of light in the sprawling manor of Willow Creek. Their story became a legend, whispered in the wind and echoing in the hearts of those who knew them, a testament to the enduring power of kindness and the unbreakable bond forged between a kind heiress and her faithful maid.